Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.5.1
+ENV GO15VENDOREXPERIMENT 1
+
+ADD . /go/src/github.com/intercom/dvara
+RUN go install github.com/intercom/dvara/cmd/dvara
+ENTRYPOINT /go/bin/dvara -addrs=$ADDRS -max_connections=1000 -max_per_client_connections=100 --port_start=17017
+
+EXPOSE 17017
```

Comment:
I don't know docker enough, but other than exposing port used to access proxy from clients, is there a need to expose mongo DB, from proxy to mongodb?

Also, mongoproxy allows a range of incoming ports ( there is a variable  
```
 -port_end int
    	end of port range (default 6010)
```